app/templates/signed-out.html

```diff
@@ -1,6 +1,6 @@
 {% extends theme('layouts/_twocol.html') %}
 
-{% block page_title %}Save Successful{% endblock %}
```

Comment:
If {{survey_title}} is /always/ meant to be here is there a way to set this in the base template so it cannot be missed? so that you just define the title to the left of the survey title in derived pages?